src/database.js

```diff
@@ -348,6 +348,12 @@ class Database extends ServiceObject {
     const reqOpts = {
       database: this.formattedName_,
     };
+
+    if (options.labels) {
+      reqOpts.session = {labels: options.labels};
+      delete options.labels;
```

Comment:
This modifies the user object, so maybe we should clone `options`?

Reply:
hey good call! just pushed a new commit